crates/subspace-farmer/src/bin/subspace-farmer/main.rs

```diff
@@ -94,6 +91,33 @@ enum Command {
     Farm(FarmingArgs),
 }
 
+fn parse_human_readable(s: &str) -> Result<u64, std::num::ParseIntError> {
+    const SUFFIXES: &[(&str, u64)] = &[
+        ("kB", 10u64.pow(3)),
+        ("k", 2u64.pow(10)),
+        ("K", 2u64.pow(10)),
+        ("KiB", 2u64.pow(10)),
+        ("mB", 10u64.pow(6)),
+        ("m", 2u64.pow(20)),
+        ("M", 2u64.pow(20)),
+        ("MiB", 2u64.pow(20)),
+        ("gB", 10u64.pow(9)),
+        ("g", 2u64.pow(30)),
+        ("G", 2u64.pow(30)),
+        ("GiB", 2u64.pow(30)),
+        ("tB", 10u64.pow(12)),
+        ("t", 2u64.pow(40)),
+        ("T", 2u64.pow(40)),
+        ("TiB", 2u64.pow(40)),
```

Comment:
`m` and `t` are not even valid suffixes. I think we only need `G` (10^9) and `T` (10^12), not sure `k` and `M` are needed. And I don't think we need the rest. If you want precise number of bytes you can still have it.